### PR TITLE
Document release process (#20)

### PR DESCRIPTION
**Pull request by willkg**
This documents the release process I'm doing with (hopefully) enough information that other people can do the same and/or find me when things go awry.

### CODE REVIEW

**Comment by willkg** [comment]
@gabrielesvelto Does this look ok? Do you think it's missing anything important?

File: README.rst [hunk 1]
```diff
@@ -105,3 +110,54 @@ PGO profile
...
+
+
+latest image
+------------
+
+To trigger building a new ``latest`` image:
+
+1. Land something in the main branch.
+
+2. wait for a bit and check https://hub.docker.com/r/mozilla/socorro-minidump-stackwalk
```
Comment (gabrielesvelto) [inline README.rst]: nit: Wait

**Review by gabrielesvelto — APPROVED** [review]
LGTM, there doesn't seem to be anything missing. Having added the python version is a nice touch because with all the semi-compatible versions floating around one never knows.

**Comment by willkg** [comment]
I clarified the Python version requirements and made the lists consistent regarding punctuation and capitalization.

Thank you!

**Pull request merged**